src/makeCalculator.js

```diff
@@ -37,7 +37,45 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    add,
+    subtract,
+    multiply,
+    divide,
+    reset,
+    operate,
+  };
+
+  function add(number) {
+    calculator.result += number;
+  }
+
+  function subtract(number) {
+    calculator.result -= number;
+  }
+
+  function multiply(number) {
+    calculator.result *= number;
```

Comment:
Use 'this.result' instead of 'calculator.result' for better readability and consistency.